### ISSUE TITLE
Work in support of CareerScope assessment tool no longer being used

### ISSUE THREAD

**Issue by RLHecht** [department-of-veterans-affairs/va.gov-team#47212]
## Issue Description
Stakeholder Tammy Hurley informed us that as of 9/27/22, CareerScope is no longer being used for Veterans. She asked us to remove all references of it from our pages. These include:

- https://www.va.gov/careers-employment/careerscope-skills-assessment/
- https://www.va.gov/education/ (under More information and resources)
- https://www.va.gov/careers-employment/ (under Manage your career)

We should also do a search to see if we find it mentioned or linked from any other pages.

Before we can retire the [CareerScope](https://www.va.gov/careers-employment/careerscope-skills-assessment/) page, Mikki told us we need to set up a redirect. She also suggested us adding a note to another page letting Veterans know this is no longer a working tool. 

We've reached out to Tammy Hurley to ask which page this should be redirected to and which page we should post the note. 

A follow-up ticket will be created for the redirect. 

## Tasks
- [x] Create note about CareerScope tool no longer working
- [x] Remove CareerScope links on education and career pages
- [x] Set up redirect ticket for CareerScope page (Randi)
- [x] Review note (Laura)
- [x] Copyedit (Megan)
- [x] Publish all changes (Megan)

## Acceptance Criteria
- [x] CareerScope information is no longer on VA.gov

**Comment by laurwill** [comment]
Documenting info from stakeholders here:
- CareerScope tool itself is not retiring, but VA is no longer supporting its use 
- 16K Veterans currently use it annually
- Replacement is O*NET Interest Profiler: https://www.onetcenter.org/IP.html

**Comment by laurwill** [comment]
Hi @mnorthuis , we're thinking we could redirect the [CareerScope page ](https://www.va.gov/careers-employment/careerscope-skills-assessment/)to this [Veteran resources](https://www.va.gov/careers-employment/veteran-resources/) page and add a new blurb along these lines:

**[O*NET Interest Profiler]**(https://www.onetcenter.org/IP.html) helps you find career options that may interest you. We’re no longer offering access to CareerScope. You can use Interest Profiler to explore career options instead.

Is this a good solution?

And I'm not able to tag Joni yet here, but just noting that we'll also loop her in about process for these types of IA questions going forward!

**Comment by heyjoni** [comment]
Let's wait for @mnorthuis to confirm while I'm ramping up, but I'm wondering if it would make more sense to create a new O*NET Interest Profiler page that explains what it is, what it's replacing, and links to the actual tool and alternatives. A new page for O*NET might be more SEO-friendly if people were to search for VA education/career tools in Google.

**Comment by laurwill** [comment]
Hi @heyjoni — thanks for raising this! Beth and I discussed this option (and I should have documented that here) and I think a new page for O*NET is still on the table for the future, but we were looking for a more immediate fix since CareerScope needs to get redirected today or tomorrow. 

Do you think redirecting to the Veteran resources page would be a good solution for the time being?

(And @bethpottsVADEPO please correct if I've missed anything here about the full-page option!)

**Comment by bethpottsVADEPO** [comment]
@laurwill @heyjoni That's correct! We'll fit into our priorities writing a page about the O*NET tool. But in the short-term we'll redirect to this resources page in Careers.

@mnorthuis and I spoke about this this morning. It's not a perfect solution, but we needed to put this temporary redirect in place quickly. Laura's going to add the new teaser/link at the top of the list of lists, so the information scent will be there since the teaser will mention CareerScope.

**Comment by mnorthuis** [comment]
From our convo this morning @bethpottsVADEPO 

Short term:
- implement a **302 redirect (temporary)** of the [CareerScope page](https://www.va.gov/careers-employment/careerscope-skills-assessment/) to the [Career resources page](https://www.va.gov/careers-employment/veteran-resources/)
- Add the O*NET tool and teaser text to the top of the Career resources page, include "CareerScope" in the teaser text
- Remove the CareeerScope option from the Education hub page and the Careers hub page
- Remove the CareerScope option from the Careers left nav
- Remove the CareerScope best bet from va.gov search results  (Mikk's to do)

Later:
- Update redirect to permanent 301 redirect once the O*NET/CareerScope R&S page is launched

**Comment by laurwill** [comment]
Hi @megzehn , 
Here are the 3 pages with content changes, to copyedit and publish today — we need these to go live before the 3pm build when PW will push the redirect live. 
- Other resources page (full copyedit): http://preview-prod.vfs.va.gov/preview?nodeId=723 
- Education hub (replaced CareerScope link in More info with link to the redirect destination): http://preview-prod.vfs.va.gov/preview?nodeId=71 
- Career hub (just removed the CareerScope link): http://preview-prod.vfs.va.gov/preview?nodeId=72

**Comment by mnorthuis** [comment]
FYI, best bet has been removed.

**Comment by laurwill** [comment]
Just noting that @megzehn copyedited and published the 3 pages, and we removed CareerScope from the nav — I'll validate that all changes are live ahead of the 3pm build that will set the redirect.

**Comment by laurwill** [comment]
I validated changes, confirmed the redirect is working, and notified the stakeholders. @RLHecht we can close this out!

Also noting this via PW for future redirect tickets: The code that manages redirects is only automatically deployed on Wednesdays at ~10am ET.  So when we have urgent timing around a redirect that doesn't align with that, we'll always need to request an off-cycle deploy.

**Comment by RLHecht** [comment]
Thanks, Laura. I'll add this info to the redirect request ticket and look at the other redirect documentation to make sure it's added there as well. Closing ticket!
